interface/themes/oe-bootstrap.scss

```diff
@@ -1,4 +1,7 @@
 /*below section is override bootstrap 3 for openemr and some new classes all prefixed with oe-*/
+input[type=checkbox], input[type=radio] {
+	margin: 4px;
+}
```

Comment:
hi @zbig01 and @d3sandoval ,
I am finding it really strange that the above changes are not "taking" in the color css scripts (only are taking in light and manila). Is this meant to happen.
To produce the css scripts, I ran `npm run build` twice

Reply:
Ok, I figured out the mystery :)
oe-bootstrap is simply not included in the colors scss, which I assume is purposeful.

Reply:
![](https://assets-cdn.github.com/images/icons/emoji/unicode/1f44d.png)